event_log.c

```diff
@@ -118,7 +118,7 @@ typedef struct __packed {
     /* u8 event[]; */
```

Comment:
Commit message:
```
To keep amount of data sent to the TPM by SINIT instruction low, .data
section is made empty by moving it's contents is made either read-only
(.rodata), or uninitialized and built at run time (.bss).
```
Looks like something went wrong while editing the sentence.  Maybe second line was meant to be:
```
section is made empty by moving its contents which is either turned read-only
```

Reply:
https://github.com/TrenchBoot/secure-kernel-loader/pull/15/commits/8a9437be860d7163b46eade6f1e2503dc9210d2b